tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -13229,6 +13249,12 @@ TEST_F(EnergyPlusFixture, UnitarySystemModel_SizingWithFans)
     thisSys.m_MaxNoCoolHeatAirVolFlow = DataSizing::AutoSize;
     thisSys.m_DesignFanVolFlowRate = DataSizing::AutoSize;
 
+    // delta temperature calculations
+    Real64 fanDT = thisSys.getFanDeltaTemp(*state, true, 1, 1.0);
+    EXPECT_NEAR(fanDT, 0.3068, 0.0001);
+    fanDT = thisSys.getFanDeltaTemp(*state, true, 0.5, 0.5);
+    EXPECT_NEAR(fanDT, 0.17615, 0.0001);
```

Comment:
Check for the new `getFanDeltaTemp` function. Here we have a multispeed fan so we expect the output to be different at a lower mass flow rate and air flow ratio.